### ISSUE TITLE
Hide wallet balance in Receive screen in GUI

### ISSUE THREAD

**Issue by heptathlon** [monero-project/monero#2721]
It is suboptimal to show a QR code which is supposed to be scanned by customers while displaying the wallet balance in the top of the left sidebar. Even if there is a button to hide the sidebar, it should be hidden by default in the Receive screen. Alternatively make it configurable whether to show the Balance at all or only in a dedicated tab (eg., My Balance tab). Maybe the History tab is already suffient for some people.

**Comment by moneromooo-monero** [comment]
Fair point, but please file in the GUI repo instead: https://github.com/monero-project/monero-core/issues
